src/tv2_afvd_showstyle/helpers/pieces/jingle.ts

```diff
@@ -49,47 +51,35 @@ export function EvaluateJingle(
 	}
 
 	if (adlib) {
```

Comment:
I would prefer if the content of the if and else statement were moved to separate function. It's clear enough that it does something if it's an adlib but it's not very clear what it does. Same goes for the else when it's not an adlib it does something else but you don't know what.